source/solvers/heat_transfer_assemblers.cc

```diff
@@ -66,11 +66,17 @@ HeatTransferAssemblerCore<dim>::assemble_matrix(
       // of the time-step
       const double tau =
         is_steady(method) ?
-          1. / std::sqrt(std::pow(2. * rho_cp * u_mag / h, 2) +
+          1. / std::sqrt(std::pow(2. * u_mag / h, 2) +
```

Comment:
I don't understand rho_cp was removed. Is it because it was unnecessary in the first place?

Reply:
Good question. It was actually ill-posed. Because it meant that tau did not have units of time anymore. It should not have been there in the first place.